robottelo/ui/rhai.py

```diff
@@ -12,3 +18,50 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def register_client_to_rhai(self, activation_key, org):
+        with VirtualMachine(distro='rhel67') as vm:
+            # Download and Install ketello-ca rpm
+            vm.install_katello_cert()
+            vm.register_contenthost(activation_key, org)
+
+            # Red Hat Access Insights requires RHEL 6/7 repo and it not
+            # possible to sync the repo during the tests,
+            # adding a file in /etc/yum.repos.d/rhel6/7.repo
+
+            rhel6_repo = conf.properties.get('insights.rhel6_repo')
```

Comment:
If the config is not provided, the value of rhel6_repo will be None, should we raise an exception?